redirects/16.0.txt

```diff
@@ -32,6 +32,11 @@ applications/sales/subscriptions/sales_flow/create_a_quotation.rst applications/
 applications/sales/subscriptions/sales_flow/renewals.rst applications/sales/subscriptions/renewals.rst                                # sales_flow/* --> *
 applications/sales/subscriptions/sales_flow/upselling.rst applications/sales/subscriptions/upselling.rst                              # sales_flow/* --> *
 
+# applications/services
+
+applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst applications/services/helpdesk/advanced/track_and_bill.rst
+applications/services/helpdesk/timesheet_and_invoice/invoice_time.rst applications/services/helpdesk/advanced/track_and_bill.rst
```

Comment:
requires an update of the links in the settings
There are two settings that linked to these pages separately. Is it okay to update them both to a single link?

https://github.com/odoo/enterprise/blob/a342fcb0e08aaeb46a6b50f11d9834bf394b1370/helpdesk/views/helpdesk_team_views.xml#L207 

https://github.com/odoo/enterprise/blob/a342fcb0e08aaeb46a6b50f11d9834bf394b1370/helpdesk/views/helpdesk_team_views.xml#L224

Reply:
Hi @jcs-odoo -They are two processes that involve many of the same setup and features but differ when it comes to whether or not the customer is billed before or after the services are completed. 

 I confirmed with the PO (AJU) that we could combine these two. She agrees that having only one document will make it easier to send to users and to link to in the settings

Reply:
Perfect :) Can you remind me, after this PR is merged, that I need to update the links in Odoo? Thanks!